cross_validation.py

```diff
@@ -1,15 +1,17 @@
-import numpy as np
+"""
+Cross validation
+"""
```

Comment:
We will either write a real documentation, or leave blank, but there is not point in adding this.

Reply:
Would the following work?

> Cross validation.
> This script performs k-fold cross validation using a Bidirictionnal GRU.

Reply:
Would the following work?

> Cross validation.
> This script performs k-fold cross validation using a Bidirectionnal GRU.